web/client-api/src/main/java/io/deephaven/web/client/api/barrage/def/ColumnDefinition.java

```diff
@@ -3,6 +3,10 @@
  */
 package io.deephaven.web.client.api.barrage.def;
 
+import io.deephaven.web.client.api.Column;
```

Comment:
This seems kind of backwards; would expect `Column` to import from `ColumnDefinition`, rather than `api.barrage.def.ColumnDefinition` importing from higher up on the package chain. Somewhat circular.

Reply:
Well, Column is the combined data from multiple ColumnDefinitions, so it doesnt have a single ColumnDefinition backing it.